leobot_control/config/gazebo/leobot_control.yml

```diff
@@ -0,0 +1,8 @@
+joint_state_controller:
+  type: joint_state_controller/JointStateController
+  publish_rate: 50
+
+joint_position_controller:
```

Comment:
which joint position controller? :) We will have controller for a few joints. Please specify that it is head.